src/mmw/js/src/app.js

```diff
@@ -56,14 +56,15 @@ var App = new Marionette.Application({
     },
 
     showLoginModal: function() {
-        new userViews.LoginModalView({
+        this.loginModal = new userViews.LoginModalView({
```

Comment:
Consider keeping the naming convention consistent and calling this `loginView` or `loginModalView`, then adding a `show` and `hide` method on it so we don't have to manually manipulate the `$el.modal`